index.js

```diff
@@ -98,20 +99,20 @@ app.use((req, res, next) => {
 
 // middleware function to check for logged-in users
 var sessionChecker = (req, res, next) => {
-    if (req.session.user && req.cookies.user_sid) {
-        res.redirect('/dashboard');
+    if (!req.session.user && !req.cookies.user_sid) {
```

Comment:
I think that this should be an OR `||` instead of an AND `&&`. Since if they are missing either of these things, we want them to sign in again.

In general, the opposite of `A && B` is `!A || !B`.

Reply:
I had actually made a note to ask about that and then forgot 😅